src/githubHelper.ts

```diff
@@ -1167,6 +1175,14 @@ export class GithubHelper {
       );
     }
 
+    // Store usernames found in the text
+    const matches: Array<string> = str.match(usernameRegex);
+    if (matches && matches.length > 0) {
+      for (const username of matches) {
+        this.users.add(username.substring(1));
+      }
+    }
```

Comment:
Here we can also replace non-mapped users by the username minus `@` symbol, so we do not tag anyone who does not belong to the project.